Smartpad-iOS/Smartpad-iOS/Logic/ConnectionManager.swift

```diff
@@ -68,6 +68,26 @@ class ConnectionManager:NSObject, MCSessionDelegate, MCNearbyServiceAdvertiserDe
     func stopHosting(){
         advertiser?.stopAdvertisingPeer()
     }
+    
+    func unpairDevice(){
+        guard let p2pSession = p2pSession else {
+            return
+        }
+        
+        let connData = ConnectionData()
+        connData.setSelectedPeer(name: "")
+        p2pSession.disconnect()
+        advertiser?.stopAdvertisingPeer()
+  
```

Comment:
Add: 
                mainVC.connStatus = ConnStatus.Unpaired
                 self.mainVC.updateConnInfoUI()

Otherwise the main view won't update to show that we are unpaired

Reply:
Try clicking unpaired on iOS, you will see the view is not updated to reflect that its unpaired